optaplanner-webexamples/pom.xml

```diff
@@ -34,41 +35,81 @@
     </testResources>
     <plugins>
       <plugin>
-        <artifactId>maven-dependency-plugin</artifactId>
+        <groupId>org.codehaus.cargo</groupId>
+        <artifactId>cargo-maven2-plugin</artifactId>
+        <configuration>
+          <container>
+            <containerId>wildfly9x</containerId>
+            <type>installed</type>
+            <artifactInstaller>
+              <groupId>org.wildfly</groupId>
+              <artifactId>wildfly-dist</artifactId>
+              <version>${version.org.wildfly}</version>
+            </artifactInstaller>
+          </container>
+          <configuration>
+            <properties>
+              <cargo.jvmargs>-Xms512m -Xmx1024m -XX:MaxPermSize=512m</cargo.jvmargs>
+              <cargo.port.offset>10000</cargo.port.offset>
+              <cargo.pingUrl>${deployable.base.url}</cargo.pingUrl>
+            </properties>
+          </configuration>
+          <deployables>
+            <deployable>
+              <groupId>org.optaplanner</groupId>
+              <artifactId>optaplanner-webexamples</artifactId>
+              <type>war</type>
+              <pingURL>${deployable.base.url}</pingURL>
+            </deployable>
+          </deployables>
+
+        </configuration>
+        <executions>
+          <execution>
+            <id>start-container</id>
+            <phase>pre-integration-test</phase>
+            <goals>
+              <goal>start</goal>
```

Comment:
Yes, as mentioned in the PR description, you can't run/debug the tests using single click in IDE (at least now right now, it would require some plugin for the IDE).